### PR TITLE
Add pokemon details in Android.

### PR DESCRIPTION
**Pull request by jimibilibob**
- Fill pokemon detail screen data in Android.
- Fix the pokeball in the iOS pokemon list.
- Fix the types in pokemon detail screen (white color).
- Remake Pokemon Evolution Chain cells in Android.

[pokemon_ev_chain_1.webm](https://user-images.githubusercontent.com/20176876/231166394-1443ddf4-2bd3-476c-afd8-fe1e43ba93e3.webm)

https://user-images.githubusercontent.com/20176876/231166427-42e157ca-feb5-4750-b0d9-8edc71abb74b.mp4

Simulator Screen Shot - iPhone 14 Pro Max - 2023-04-11 at 01 03 22
Simulator Screen Shot - iPhone 14 Pro Max - 2023-04-11 at 01 03 14

### CODE REVIEW

File: PokeAppAndroid/Resources/layout/fragment_info_detail.xml [hunk 1]
```diff
@@ -127,6 +129,7 @@
                         android:text="Kanto"/>
```
Comment (YawarValeriano) [inline PokeAppAndroid/Resources/layout/fragment_info_detail.xml]: use tools:text instead

File: PokeAppAndroid/Resources/layout/fragment_info_detail.xml [hunk 2]
```diff
@@ -73,6 +73,7 @@
                         android:text="Unknown"/>
```
Comment (YawarValeriano) [inline PokeAppAndroid/Resources/layout/fragment_info_detail.xml]: change all android text to use tools: text

File: PokeAppAndroid/View/PokemonEvolutionFragment.cs [hunk 3]
```diff
@@ -1,24 +1,79 @@
...
+
+        public void updateEvoutionChain(Result<EvolutionChainResponse> evolutionChain)
+        {
+            if (evolutionChain.IsFailure) return;
+            _evolutionChain = evolutionChain.Value;
+
+            var mLayoutManager = new LinearLayoutManager(RequireContext());
+            RvEvolutionChain.SetLayoutManager(mLayoutManager);
+
+            //if (viewHolder.RvTypes.ItemDecorationCount < 1)
```
Comment (YawarValeriano) [inline PokeAppAndroid/View/PokemonEvolutionFragment.cs]: remove commented code

**Review by YawarValeriano — CHANGES_REQUESTED** [review]

File: PokeAppiOS/Views/Cells/PokemonViewCell.xib [hunk 4]
```diff
@@ -125,7 +125,7 @@
     </objects>
     <resources>
         <image name="Bulbasaur" width="200" height="200"/>
-        <image name="pokeball" width="48" height="48"/>
+        <image name="Pokeball" width="206" height="208"/>
```
[the remark this answers is not in the document]
Comment (jimibilibob) [reply PokeAppiOS/Views/Cells/PokemonViewCell.xib:128]: The pokeball image is shown in last commit.

Simulator Screen Shot - iPhone 14 Pro Max - 2023-04-11 at 09 41 13

File: PokeAppAndroid/View/PokemonEvolutionFragment.cs (hunk 3 above)
Comment (jimibilibob) [reply PokeAppAndroid/View/PokemonEvolutionFragment.cs]: Removed.

File: PokeAppAndroid/Resources/layout/fragment_info_detail.xml (hunk 1 above)
Comment (jimibilibob) [reply PokeAppAndroid/Resources/layout/fragment_info_detail.xml]: Using tools now.

File: PokeAppAndroid/Resources/layout/fragment_info_detail.xml (hunk 2 above)
Comment (jimibilibob) [reply PokeAppAndroid/Resources/layout/fragment_info_detail.xml]: Using tools now.

**Review by YawarValeriano — APPROVED** [review]

**Review by rodrigoschar — APPROVED** [review]

**Review by adCBueno — APPROVED** [review]

**Pull request merged**